x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc

```diff
@@ -0,0 +1,595 @@
+[[security-basic-setup-https]]
+=== Set up basic security for the Elastic Stack plus secured HTTPS traffic
+++++
+<titleabbrev>Set up basic security plus HTTPS</titleabbrev>
+++++
+
+In a production environment, some {es} features such as tokens and
+API keys will be disabled unless you enable TLS on the HTTP layer. This
+addition layer of security ensures that all communications to and from your
+cluster are secured.
+
+When you run the `elasticsearch-certutil` tool in `http` mode, the tool asks
+several questions about how you want to generate certificates. While there are
+numerous options, the following choices result in certificates that should
+work for most environments.
+
+[[signing-certificates]]
+.Signing certificates
+****
+The first question that the `elasticsearch-certutil` tool prompts you with is
+whether you want to generate a Certificate Signing Request (CSR). Answer
+`n` if you want to sign your own certificates, or `y` if you want to sign
+certificates with a central CA.
+
+[discrete]
+===== Sign your own certificates
+
+If you want to use the CA that you created when
+<<generate-certificates,Generating the certificate authority>> answer `n` to
+the first question. You then specify the location of your CA, which the tool
+uses to sign and generate a `.p12` certificate. The steps in this procedure
+follow this workflow.
+
+[discrete]
+===== Sign certificates with a central CA
+
+If you work in an environment with a central security team, they can likely
+generate a certificate for you. Infrastructure within your organization
```

Comment:
Nit: I think there is a difference between "generate" and "sign" a certificate. If the central security team can just generate the certificate, we don't even need to use the `elasticsearch-certutil` tool at all. The tool is relevant if the security team "signs" the certificate for which we need the CSR file.